### PR TITLE
[Tests-Only] used skeleton folders more wisely in API provisioning tests

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
With this PR:
- skeleton file/folders are used more wisely on apiProvisioning* tests

## Related Issue

- Part of https://github.com/owncloud/QA/issues/658

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiProvisioning-v1/editUser.feature [hunk 1]
```diff
@@ -4,20 +4,20 @@ Feature: edit users
   I want to be able to edit user information
   So that I can keep the user information up-to-date
 
-  Background:
-    Given using OCS API version "1"
+#  Background:
+#    Given using OCS API version "1"
```
Comment (Talank) [inline tests/acceptance/features/apiProvisioning-v1/editUser.feature:8]: Why commenting it? And why not deleting instead of commenting? :thinking:

**Review by Talank — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/apiProvisioning-v1/editUser.feature (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/apiProvisioning-v1/editUser.feature:8]: just for a test purpose! i've reverted it.

**Review by Talank — COMMENTED** [review]
LGTM

**Review by Talank — APPROVED** [review]

**Review by dpakach — COMMENTED** [review]
Lgtm :+1:

**Review by individual-it — APPROVED** [review]

File: tests/acceptance/features/apiProvisioning-v1/enableUser.feature [hunk 2]
```diff
@@ -91,12 +91,12 @@ Feature: enable user
...
   @notToImplementOnOCIS
   Scenario: Making a web request with an enabled user
-    Given user "Alice" has been created with default attributes and small skeleton files
+    Given user "Alice" has been created with default attributes and without skeleton files
     When user "Alice" sends HTTP method "GET" to URL "/index.php/apps/files"
     Then the HTTP status code should be "200"
 
   Scenario: normal user should not be able to enable himself
-    Given these users have been created with default attributes and small skeleton files:
+    Given these users have been created with default attributes and without skeleton files:
```
Comment (jasson99) [inline tests/acceptance/features/apiProvisioning-v1/enableUser.feature:99]:
```suggestion
    Given user "Alice" been created with default attributes and without skeleton files:
```

File: tests/acceptance/features/apiProvisioning-v2/deleteUser.feature [hunk 3]
```diff
@@ -65,7 +65,7 @@ Feature: delete users
     And user "Brian" should exist
 
   Scenario: administrator deletes another admin user
-    Given these users have been created with default attributes and small skeleton files:
+    Given these users have been created with default attributes and without skeleton files:
```
Comment (jasson99) [inline tests/acceptance/features/apiProvisioning-v2/deleteUser.feature:68]:
```suggestion
    Given user "another-admin" has been created with default attributes and without skeleton files:
```

File: tests/acceptance/features/apiProvisioning-v2/enableUser.feature [hunk 4]
```diff
@@ -92,13 +92,13 @@ Feature: enable user
...
   Scenario: Making a web request with an enabled user
-    Given user "Alice" has been created with default attributes and small skeleton files
+    Given user "Alice" has been created with default attributes and without skeleton files
     When user "Alice" sends HTTP method "GET" to URL "/index.php/apps/files"
     Then the HTTP status code should be "200"
 
   @notToImplementOnOCIS
   Scenario: normal user should not be able to enable himself
-    Given these users have been created with default attributes and small skeleton files:
+    Given these users have been created with default attributes and without skeleton files:
```
Comment (jasson99) [inline tests/acceptance/features/apiProvisioning-v2/enableUser.feature:101]:
```suggestion
    Given user "Alice" has been created with default attributes and without skeleton files:
```

File: tests/acceptance/features/apiProvisioningGroups-v1/getUserGroups.feature [hunk 5]
```diff
@@ -130,7 +130,7 @@ Feature: get user groups
 
 
   Scenario: normal user gets his/her groups
-    Given these users have been created with default attributes and small skeleton files:
+    Given these users have been created with default attributes and without skeleton files:
       | username |
       | Alice    |
```
Comment (jasson99) [inline tests/acceptance/features/apiProvisioningGroups-v1/getUserGroups.feature:135]:
```suggestion
    Given user "Alice" has been created with default attributes and without skeleton files:
```

File: tests/acceptance/features/apiProvisioningGroups-v2/getUserGroups.feature [hunk 6]
```diff
@@ -131,7 +131,7 @@ Feature: get user groups
 
 
   Scenario: normal user gets his/her groups
-    Given these users have been created with default attributes and small skeleton files:
+    Given these users have been created with default attributes and without skeleton files:
       | username |
       | Alice    |
```
Comment (jasson99) [inline tests/acceptance/features/apiProvisioningGroups-v2/getUserGroups.feature:136]:
```suggestion
    Given user "Alice" has been created with default attributes and without skeleton files:
```

**Review by jasson99 — COMMENTED** [review]
And also resolve the conflicts

**Pull request merged**